addons/semicolon/semicolon.js

```diff
@@ -0,0 +1,14 @@
+export default async function ({ addon, global, console }) {
+  if (document.querySelector("body #signature")) {
+    console.log("Signature already exists!");
+  } else {
+    if (addon.tab.editorMode === "editor") {
+      console.log("In editor!");
+    } else {
+      var semicolon = document.createElement("p");
+      semicolon.textContent = ";";
+      semicolon.classList.add("semicolon");
+      document.body.appendChild(semicolon);
+    }
```

Comment:
This looks like going into a project in player mode, then into editor mode, will still show the semicolon. Can you use `addon.tab.addEventListener("urlChange", () => {})` to apply `display: none;` to the semicolon if the user went into editor or fullscreen mode?